src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,21 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
```

Comment:
Ensure that the `hibernate.cfg.xml` file is present in the resources directory. This file is crucial for configuring Hibernate and initializing the `SessionFactory`.